packages/core/src/hooks.ts

```diff
@@ -266,7 +267,11 @@ function getStateHooks(useConnector: UseBoundStore<Web3ReactStore>) {
     return useConnector(ACTIVATING)
   }
 
-  return { useChainId, useAccounts, useIsActivating }
+  function useHasError():  Web3ReactState['error'] {
```

Comment:
This should read `useError()`, because you are returning the error object and not a boolean.

Reply:
sg done